src/deploy/functions/tasks.ts

```diff
@@ -196,6 +216,14 @@ export function deleteFunctionTask(params: TaskParams, fn: backend.FunctionSpec)
   };
 }
 
+export async function setConcurrency(name: string, concurrency: number) {
```

Comment:
This probably doesn't need to be exported, unless you just want to do so for writing a unit test.